java/core/src/test/org/apache/orc/impl/TestRecordReaderUtils.java

```diff
@@ -177,6 +178,37 @@ public void execute() throws Throwable {
     assertTrue(dis.isAllReleased());
   }
 
+  @Test
+  public void testBufferChunkOffsetExceedsMaxInt() {
+    List<long[]> mockData = Arrays.asList(
+            new long[]{15032282586L, 15032298848L}
+            , new long[]{15032298848L, 15032299844L}
```

Comment:
nit. Could you put `,` at the end of the previous line instead of the first of line?